### PR TITLE
Add missing openssl dependency to rgi

### PR DESCRIPTION
**Pull request by innovate-invent**
RGI has an unspecified openssl=1.0 dependency. This adds that dependency.
https://github.com/arpcard/rgi/issues/74

----

> Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences**
      (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

> Everyone has access to the following BiocondaBot commands, which can be given in a comment:
>
>  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
>  * `@BiocondaBot please add label` will add the `please review & merge` label.
>  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
>
> For members of the Bioconda project, the following command is also available:
>
>  * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.
>
> If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

File: recipes/rgi/meta.yaml [hunk 1]
```diff
@@ -61,6 +61,7 @@ requirements:
     - bamtools >=2.5.1
     - bedtools >=2.28.0
     - bwa >=0.7.17
+    - openssl ==1.0
```
Comment (npavlovikj) [inline recipes/rgi/meta.yaml]: Please use `openssl 1.0.*` instead. If you see the log info, you will see that openssl 1.1.1d is installed due to the incorrect syntax.

Comment (innovate-invent) [reply recipes/rgi/meta.yaml]: That's odd, I wonder what the difference between my command line is from the CI

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

**Comment by dpryan79** [comment]
The openssl error has nothing to do with rgi, but rather getting an old blast installation. I'll see if I can correct the recipe to enable more recent (and therefore properly pinned to openssl) blast versions.

File: recipes/rgi/meta.yaml [hunk 2]
```diff
@@ -61,6 +61,7 @@ requirements:
     - bamtools >=2.5.1
     - bedtools >=2.28.0
     - bwa >=0.7.17
+    - openssl ==1.0.*
```
Comment (dpryan79) [inline recipes/rgi/meta.yaml]: This won't really solve the problem, the solution is to rebuild blast 2.6.0.

Comment (dpryan79) [reply recipes/rgi/meta.yaml]: Actually, the version of blast this is using is wrong.

Comment (innovate-invent) [reply recipes/rgi/meta.yaml]: There is a comment in the file saying it requires that exact version of blast

Comment (dpryan79) [reply recipes/rgi/meta.yaml]: On the readme file on github it mentions 2.9.0, so I went with that in my PR.

**Pull request closed without merging**